views/dashboard-widgets/limits-and-quotas.php

```diff
@@ -115,21 +115,21 @@
 
 			<?php esc_html_e('Unique Visits', 'wp-multisite-waas'); ?>
 
-			<?php echo wu_tooltip(sprintf(__('Next Reset: %s', 'wp-multisite-waas'), date_i18n(get_option('date_format', 'd/m/Y'), strtotime('last day of this month')))); ?>
+			<?php echo wp_kses_post(wu_tooltip(sprintf(esc_html__('Next Reset: %s', 'wp-multisite-waas'), date_i18n(get_option('date_format', 'd/m/Y'), strtotime('last day of this month'))))); ?>
```

Comment:
wp_tooltip is safe. lets not do extra escaping and I think wp_kses_post actually causes problems
```suggestion
			<?php echo wu_tooltip(sprintf(esc_html__('Next Reset: %s', 'wp-multisite-waas'), date_i18n(get_option('date_format', 'd/m/Y'), strtotime('last day of this month')))); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```